File: backend/src/main/java/cohappy/backend/model/dto/request/PatchChatUsersDTO.java
package cohappy.backend.model.dto.request;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class PatchChatUsersDTO {
    private String chatCode;
    private List<String> usersCode;
    private PatchChatUsersOperationEnum operation;
}

File: frontend/app/src/main/java/cohappy/frontend/model/dto/request/PatchChatUsersDTO.kt
package cohappy.frontend.model.dto.request

import cohappy.frontend.model.dto.PatchChatUsersOperationEnum
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class PatchChatUsersDTO(
    val chatCode: String,
    var usersCode: List<String>? = null,
    val operation: PatchChatUsersOperationEnum
)
